site/content/en/blog/releases/1.24.0.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "1.24.0"
+linkTitle: "1.24.0"
+date: "2022-06-21"
+---
+This is the 1.24.0 Release of Agones.
+
+This release has lots of important bug fixes and documentation. This includes:
+* ARM64 image for the simple-game-server example.
+* Fix for scaling a fleet when a game server is allocated from an older version.
+* Fix for sdkServer.logLevel nil in Gameserver in local-mode.
+* sdkServer.logLevel in Gameserver config can now be used in local mode.
+
+We also believe we've fixed all the issues with running on ARM64 clusters with this release, so please give a try (especially while we are in RC and can potentially fix things before stable next week).
```

Comment:
```suggestion
We also believe we've fixed all the issues with running on ARM64 clusters with this release, so please give a try/
```

Since we aren't in RC anymore.